src/quark-core/src/QuarkStateManager.sol

```diff
@@ -15,41 +15,42 @@ contract QuarkStateManager {
     event NonceSubmitted(address wallet, bytes32 nonce, bytes32 replayToken);
 
     /// @notice Represents the unclaimed bytes32 value.
-    bytes32 public constant CLAIMABLE_TOKEN = bytes32(uint256(0));
+    bytes32 public constant UNCLAIMED_TOKEN = bytes32(uint256(0));
 
     /// @notice A token that implies a Quark Operation is no longer replayable.
     bytes32 public constant NO_REPLAY_TOKEN = bytes32(type(uint).max);
 
     /// @notice Mapping from nonces to last used replay token.
-    mapping(address wallet => mapping(bytes32 nonce => bytes32 lastToken)) public nonceTokens;
+    mapping(address wallet => mapping(bytes32 nonce => bytes32 lastToken)) public nonceReplayTokens;
 
     /**
      * @notice Returns the nonce token (last replay token) for a given nonce. For finalized scripts, this will be `uint256(-1)`. For unclaimed nonces, this will be `uint256(0)`. Otherwise, it will be the next value in the replay chain.
```

Comment:
nit: "Otherwise, it will be the next value in the replay chain." is technically not correct. It's really the last used value in the replay chain. The "next value" is a secret 😉